src/OrbitGl/TimeGraph.cpp

```diff
@@ -519,8 +514,8 @@ void TimeGraph::PrepareBatcherAndUpdatePrimitives(PickingMode picking_mode) {
   CaptureViewElement::UpdatePrimitives(primitive_assembler_, text_renderer_static_, min_tick,
                                        max_tick, picking_mode);
 
-  if (!absl::GetFlag(FLAGS_enforce_full_redraw)) {
-    update_primitives_requested_ = false;
+  if (absl::GetFlag(FLAGS_enforce_full_redraw)) {
+    update_primitives_requested_ = true;
```

Comment:
I found a bit inconsistent that here we are only setting the update_primitives_requested_ flag but in RequestUpdate we are setting both. I think that this is the correct way, and we should change the other one (only setting update_primitives flag in RequestUpdate).

Reply:
True. I've changed this to call `RequestUpdate()` instead, I think that's more consistent and easier to read.